FAQ.md

```diff
@@ -1,30 +1,32 @@
-This FAQ attempts to address common questions and concerns around the JS Foundation's [Code of Conduct][]. If you still have questions after reading it, please feel free to [contact us][].
+# Frequently Asked Questions
+
+This FAQ attempts to address common questions and concerns around TC39's [Code of Conduct][]. If you still have questions after reading it, please feel free to [contact us][].
 
 ### Why have you adopted a Code of Conduct?
 
-We think the JS Foundation community is awesome. If you're familiar with the JS Foundation community, you'll probably notice that the Code basically matches what we already do. Think of this as documentation: we're taking implicit expectations about behavior and making them explicit.
+We think the TC39 community is awesome. If you're familiar with the TC39 community, you'll probably notice that the Code basically matches what we already do. Think of this as documentation: we're taking implicit expectations about behavior and making them explicit.
```

Comment:
> We think the TC39 community is awesome

s/TC39 community/JS community/

> If you're familiar with the TC39 community, you'll probably notice that the Code basically matches what we already do.

I'm not sure this applies in here. We can rephrase this to: "If you're familiar with the TC39, you'll probably notice we are making progressive efforts to promote and improve diversity and inclusion to match the Code and our high technical experience with standards work".